minimemcached_test.go

```diff
@@ -18,6 +19,34 @@ import (
 var cfg = &Config{
 	Port: 0,
 }
+var port uint16
+var clk = clock.NewMock()
+var mc *memcache.Client
+var mm *MiniMemcached
```

Comment:
can you please group variable declarations?

it's good to follow uber go style guide.

https://github.com/uber-go/guide/blob/master/style.md#group-similar-declarations